massdatasets.py

```diff
@@ -41,22 +46,47 @@ def add_song(self, artist, title, style, audio_filepath, **kwargs):
              **kwargs}
         )
 
-    def to_pandas_df(self):
+    def to_pandas_df(self, include_features=True):
         '''
         Compiles the yaml document to a pandas DataFrame.
         audio_filepath are complete (prefixed by base_path).
         '''
 
-        frame = pd.DataFrame(columns=self.songs[0].keys())
+        long_format = True
```

Comment:
What is the meaning of `long_format`? You use the term also in a comment in line 72.

Reply:
Yeah I should add this as a parameter and include documentation.
But anyway: https://sejdemyr.github.io/r-tutorials/basics/wide-and-long/